x-pack/plugins/canvas/public/style/index.scss

```diff
@@ -44,7 +44,7 @@
 @import '../components/remove_icon/remove_icon';
 @import '../components/rotation_handle/rotation_handle';
 @import '../components/shape_preview/shape_preview';
-@import '../components/shape_picker_mini/shape_picker_mini';
+@import '../components/shape_picker/shape_picker';
```

Comment:
Were you planning to replace `mini` with `popover` here too?

Reply:
I deleted the css file for `mini` (popover) entirely and created a new one for the picker itself.